rhobs/alerting/data_plane/prometheus.pipeline_alerts.yaml

```diff
@@ -47,3 +47,48 @@ spec:
             alert_team_handle: <!subteam^S03GF42RBE2>
             team: pipelines
             runbook_url: TBD
+        - alert: CorePipelineControllerRepeatedRestarts
+          expr: |
+            sum by (source_cluster) (increase(kube_pod_container_status_restarts_total{namespace="openshift-pipelines", pod=~"tekton-pipelines-controller-.*"}[2m])) > 0
```

Comment:
I have a concern here about the time window. I am afraid that there would be false positive alerts. IIRC, there are 2 controller pods in a cluster. If these both are restarted while doing upgrade/deployment, I shouldn't get PD alert in that case. 

Can I ask if it would be better to adjust the time window & thresholds? 

Also, does the severity level correct for all envs? I am just wondering if this is would be applied for staging clusters and both envs (prod & stage) have same severity levels.

Reply:
Can you give me suggestions for the time windows and thresholds @zanssa ?
You are correct about 2 controller pods existing in the konflux clusters.

WRT varying severity AFAIK I cannot distinguish between stage and prod with these alertrs and their severity.  Do I have that correct @amisstea ?

Reply:
The severity levels are unchanged across envs for all our alerts right now. We haven't used them for anything other than display purposes so far.

Reply:
what is your thoughts if we modify the time window for `increase` to be 5m instead of 2m ? 

Before increasing the threshold, I would like to test this and for now I am not aware of an env where I can play with the query. It would be easy to modify that later and I don't wanna block merging this. 

Thanks a lot for working and coordinating on this!

Reply:
sure @zanssa I'll tweak the timewindow to `5m` and see if I have to adjust the test cases any

The query is now possible from https://grafana.stage.devshift.net/d/rhtap-slos/rhtap-slos?orgId=1&refresh=1h if that is place where you are comfortable performing test queries

Also, if you have login credentials to the konflux clusters themselves , you can go to the OCP console's Observe tab and try there.

Reply:
updated timewindow in a separate commit for now @zanssa  ... ptal when you get the chance

Reply:
You can also use the explore view from AppSRE's grafana. Here's the prod instance (more stable): https://grafana.app-sre.devshift.net/explore

Select either of the 'rhtap-observatorium-production` or `rhtap-observatorium-stage` datasources depending on the env you want to query.

Reply:
Cool. Thank you @gabemontero & @amisstea. This is helpful. I'll definitely explore these options later. 

While exploring, I don't really wanna keep this on hold, and since we have increased the time window for `increase`, we can merge this and modify later if needed.

Reply:
OK @zanssa I have removed the WIP prefix, though it does not appear to be enabled for this repo (and I can't add labels).

Reply:
Feel free to resolve the conversations as I can't do that to unblock the merge.

Reply:
@amisstea do I need to have a specific permission to be able to use the explore view from AppSRE's grafana? If I tried to open the link, I would get `Login failed. Failed to get token from provider` - I clicked on sign in with SSO, and I got the same error msg.